lsp/nls/src/linearization/mod.rs

```diff
@@ -410,6 +417,10 @@ impl Linearizer for AnalysisHost {
                     meta,
                 },
             )
+            .map(|item| LinearizationItem {
+                ty: transform_wildcard(wildcard_vars.clone(), item.ty),
```

Comment:
Do you really need to clone the whole array? `transform_wildcard` should rather take an immutable reference to `wildcard_vars`.